extensions/adobe/experience/marketo/marketo-person-contact.schema.json

```diff
@@ -0,0 +1,204 @@
+        },
+        "marketo:customer": {
+          "title": "Customer Indicator",
+          "description": "Customer indicator.",
+          "type": "boolean"
+        },
+        "marketo:employee": {
+          "title": "Employee Indicator",
+          "description": "Employee indicator.",
+          "type": "boolean"
+        },
+        "marketo:lead": {
+          "title": "Lead Indicator",
+          "description": "Lead indicator.",
+          "type": "boolean"
+        },
+        "marketo:partner": {
+          "title": "Partner Indicator",
+          "description": "Partner indicator.",
+          "type": "boolean"
+        },
+        "marketo:private": {
+          "title": "Private Indicator",
+          "description": "Private indicator.",
+          "type": "boolean"
+        },
+        "marketo:marketoAccountID": {
+          "title": "Marketo Account id",
+          "description": "Marketo account id.",
+          "type": "string"
+        },
+        "marketo:marketingSuspended": {
+          "title": "Marketing Suspended",
+          "description": "Marketing suspended.",
+          "type": "boolean"
+        },
+        "marketo:marketingSuspendedCause": {
+          "title": "Marketing Suspended Cause",
+          "description": "Marketing suspended cause.",
+          "type": "string"
+        },
+        "marketo:notes": {
+          "title": "Notes",
+          "description": "Notes.",
+          "type": "string"
+        },
+        "marketo:orgID": {
+          "title": "Organization ID",
+          "description": "Organization id.",
+          "type": "string"
+        },
+        "marketo:primaryLeadInterestID": {
+          "title": "Primary Lead Interest Id",
+          "description": "Primary lead interest id.",
+          "type": "boolean"
+        },
+        "marketo:registerSourceInfo": {
+          "title": "Register Source Info",
+          "description": "Register source info.",
+          "type": "string"
+        },
+        "marketo:registerSourceType": {
+          "title": "Register Source Type",
+          "description": "Register source type.",
+          "type": "string"
+        },
+        "marketo:salesContactOwnerID": {
+          "title": "Sales Contact Owner Id",
+          "description": "Sales contact owner id.",
+          "type": "string"
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/profile-person-details"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/profile-personal-details"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/profile-work-details"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/profile-owning-entities"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/profile-contact-crm"
+    },
+    {
+      "$ref": "https://ns.adobe.com/xdm/context/optinout"
+    },
+    {
+      "$ref": "#/definitions/marketo-person-contact"
+    }
+  ],
```

Comment:
required Marketo:contactID and Marketo:AccountID

Reply:
As per industry standards, in a massive data lake and data warehouse, we should not impose hard referential constraints on foreign keys as certain data might flow before the foreign key is populated in the source. enforcing the primary key definitely makes sense but enforcing the foreign key might cause data lags. Please suggest if you still want me to add Account to the required ?

Reply:
What does required mean, It just implies that the property cannot be NULL? It is not enforcing the validation semantics to cross-verify whether the parent key exists as part of the enablement validation.